features/spaces/invitations.feature

```diff
@@ -54,11 +54,15 @@ Feature: Spaces: Invitations
     When a Space Owner invites a new Space Member via SMS
     Then an Invitation is sent to that SMS
 
-  @built @andromeda @unimplemented-steps
-  Scenario: Ignoring invitations
-    Given an Invitation to the "System Test" Space is sent by Space Owner "space-owner@example.com"
-      | name | email                       |
-      | Aang | aang-the-avatar@example.com |
-    When the Invitation to "aang-the-avatar@example.com" is Ignored
-    Then the Invitation to "aang-the-avatar@example.com" for the "System Test" Space has a status of "ignored"
-    And no further Invitations can be sent to "aang-the-avatar@example.com" for the "System Test" Space
+  @unimplemented-steps @unstarted
+  Scenario: Revoking a Pending Invitation
+    Given an Invitation to "invitee-a@example.com" for the "System Test" Space is sent by Space Owner "space-owner@example.com"
+    When the Invitation to "invitee-a@example.com" for the "System Test" Space is Revoked by Space Owner "space-owner@example.com"
+    Then the Invitation to "invitee-a@example.com" for the "System Test" Space has a status of "revoked"
+    Then the Invitation to "invitee-a@example.com" for the "System Test" Space cannot be Accepted
+
+  @unimplemented-steps @unstarted
+  Scenario: Revoking Accepted Invitations
```

Comment:
This is technically built, by hiding the "revoke" button unless the invitation is "pending". So maybe update the tags on this scenario and maybe even tweak the scenario description to clarify that this works by hiding the controls.

Reply:
Thanks fixed in 467c4fa